# Array of detected ArUco markers
std_msgs/Header header     # ROS Header (stamp, frame_id for the array itself)
ArucoMarker[] markers